cockatrice/src/carddatabasemodel.h

```diff
@@ -12,7 +12,7 @@ class FilterTree;
 class CardDatabaseModel : public QAbstractListModel {
     Q_OBJECT
 public:
-    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, CMCColumn };
+    enum Columns { NameColumn, SetListColumn, ManaCostColumn, PTColumn, CardTypeColumn, ColorColumn };
```

Comment:
Is there a cmc column at all? Have there ever been?

I even reinstalled the newest master build (e81a6d4) to check... there is no such column in the deck editor. :)